azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/PartitionEndOfStreamInformation.java

```diff
@@ -25,7 +25,7 @@ public String getPartitionId() {
         return this.partitionId;
     }
 
-    public long getLastSequenceNumber() {
+    public long getLastEnqueuedSequenceNumber() {
```

Comment:
I think the fact that the event has been enqueued is obvious. `lastSequenceNumber` seems clear to me, and the extra bit is unneeded verbosity. 

At the end of the day, I'm fine with either, but my preference would be that all other clients get updated to `getLastSequenceNumber`.

Reply:
on a stream first and last are relative - last depends on whats your first - `lastEnqueued` is an attempt to make it explicit... 

all other clients also have getLastSequenceNumber only - just that other properties (like offset, and timestamp) in this class has `Enqueued` in them - it felt a little different for this property to not have it!